src/FileUpload.js

```diff
@@ -0,0 +1,115 @@
+
+import React, {useState} from "react"
+
+// Import from "@inrupt/solid-client"
+import {
+  saveFileInContainer,
+  getSourceUrl
+} from '@inrupt/solid-client';
+
+// Import from "@inrupt/solid-client-authn-browser"
+import {
+  fetch
+} from '@inrupt/solid-client-authn-browser';
+
+
+function FileUpload(props)
+{
+	let currentPath = props.currentPath;
+	let [selectedFiles, setSelectedFiles] = useState([]);
+
+
+	function upload() 
+	{
+	    if(selectedFiles.length != 0)
+	    {
+	        console.log("selected file(s):");
+	        for (let file of selectedFiles)
+	        {
+	            console.log(file.name);
+	        }
+	    }
+
+	    console.log("uploading files ...");
+	    for (let file of selectedFiles)
+	    {
+	      	placeFileInContainer(file, currentPath);
+	    }
+	}
+
+
+	/**
+	 * Upload file into the targetContainer.
+	 * @param  {[type]} file               [description]
+	 * @param  {[type]} targetContainerURL [description]
+	 * @return {[type]}                    [description]
+	 */
+	async function placeFileInContainer(file, targetContainerURL) {
+	  try 
+	  {
+  		const savedFile = await saveFileInContainer(
```

Comment:
Indent/format properly (VS code can do that for you)

Reply:
done